zetaclient/chains/sui/observer/outbound.go

```diff
@@ -134,6 +133,9 @@ func (ob *Observer) VoteOutbound(ctx context.Context, cctx *cctypes.CrossChainTx
 		return errors.Wrap(err, "unable to parse gas used")
 	}
 
+	// TODO : Update the effective gas limit and gas used.
+	// Setting the effective gas limit to the gas used disables the gas stability pool funding
+	// https://github.com/zeta-chain/node/issues/4015
 	// Create message
 	msg := cctypes.NewMsgVoteOutbound(
```

Comment:
Even with this fix, can we disable the gas stability pool funding for Sui as it will not be necessary?

This is a extra safety issue, we can test the fix on `develop` in the future